app/javascript/articles/components/ReactionsCount.jsx

```diff
@@ -15,24 +15,26 @@ export const ReactionsCount = ({ article }) => {
     </svg>
   );
 
-  return (
-    <Button
-      variant="ghost"
-      size="s"
-      contentType="icon-left"
-      url={article.path}
-      icon={reactionsSVG}
-      tagName="a"
-    >
-      <span title="Number of reactions">
-        {totalReactions}
-        <span className="hidden s:inline">
-          &nbsp;
-          {`${totalReactions == 1 ? 'reaction' : 'reactions'}`}
+  if (totalReactions > 0) {
```

Comment:
I might be wrong here but I think you can return nothing immediately at the top of the function if the count is 0 because the rest of the component is just about rendering 1 or plus as the count

Reply:
makes sense 😄  I will modify